libfwupdplugin/fu-bluez-device.c

```diff
@@ -433,6 +505,80 @@ fu_bluez_device_write (FuBluezDevice *self,
 	return TRUE;
 }
 
+/**
+ * fu_bluez_device_notify_start:
+ * @uuid: The UUID, e.g. `00cde35c-7062-11eb-9439-0242ac130002`
+ * @error: A #GError, or %NULL
+ *
+ * Enables notifications for property changes in a UUID (StartNotify
+ * method).
+ *
+ * Returns: %TRUE if the method call completed successfully.
+ *
+ * Since: 1.5.8
+ **/
+gboolean
+fu_bluez_device_notify_start (FuBluezDevice *self, const gchar *uuid, GError **error)
+{
+	FuBluezDeviceUuidHelper *uuid_helper;
+	g_autoptr(GVariant) retval = NULL;
+
+	uuid_helper = fu_bluez_device_get_uuid_helper (self, uuid, error);
+	if (uuid_helper == NULL)
+		return FALSE;
+	if (uuid_helper->signal_id > 0)
+		return TRUE;
```

Comment:
ohh i think we need to remove these two lines now -- the signal is unconditionally wired up at proxy creation, so these can't fail.